### PR TITLE
Add suppliers daily sync

### PR DESCRIPTION
**Pull request by EvaPaunovaPhwk**

### CODE REVIEW

File: src/controllers/IntegrationEventsController.ts [hunk 1]
```diff
@@ -104,4 +108,100 @@ export class IntegrationEventsController {
...
+        const eventName = 'external_integrations_daily_sync';
+
+        const logger = req.requestContext.logger.child({
+            payhawkEventCorrelationId: uuid(),
+            source,
+        });
+
+        logger.info`${eventName} received`;
+
+        if (source !== Events.AccountingIntegration.MicrosoftDynamicsBusinessCentral) {
```
Comment (vassildinev) [inline src/controllers/IntegrationEventsController.ts:127]:
```suggestion
        if (source !== Events.AccountingIntegration.Xero) {
```

maybe?

File: src/controllers/IntegrationEventsController.ts [hunk 2]
```diff
@@ -104,4 +108,100 @@ export class IntegrationEventsController {
...
+
+        const logger = req.requestContext.logger.child({
+            payhawkEventCorrelationId: uuid(),
+            accountId,
+            source,
+        });
+
+        logger.info`${eventName}  received`;
+
+        if (source !== Events.AccountingIntegration.MicrosoftDynamicsBusinessCentral) {
```
Comment (vassildinev) [inline src/controllers/IntegrationEventsController.ts:157]:
```suggestion
        if (source !== Events.AccountingIntegration.Xero) {
```

maybe?

File: src/controllers/IntegrationEventsController.ts (hunk 1 above)
Comment (EvaPaunovaPhwk) [reply src/controllers/IntegrationEventsController.ts]: Thank you, fixed

File: src/controllers/IntegrationEventsController.ts (hunk 2 above)
Comment (EvaPaunovaPhwk) [reply src/controllers/IntegrationEventsController.ts]: Thank you, fixed

**Review by vassildinev — APPROVED** [review]

**Pull request merged**